ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-message-form-dialog/gio-ps-flow-message-form-dialog.component.ts

```diff
@@ -21,28 +21,28 @@ import { cloneDeep, uniqueId } from 'lodash';
 import { FlowVM } from '../../gio-policy-studio.model';
 import { ChannelSelector, ConditionSelector } from '../../models';
 
-export type GioPolicyStudioFlowFormDialogData = {
+export type GioPolicyStudioFlowMessageFormDialogData = {
   flow?: FlowVM;
   entrypoints?: string[];
 };
 
 export type GioPolicyStudioFlowFormDialogResult = FlowVM | false;
```

Comment:
It's strange to see the proxy dialog using this type from message dialog. 💭 
what about creating an internal model in the `componant/gio-ps-flow-form-dialog.model.ts` ?